python/note_input_widget/widget.py

```diff
@@ -444,7 +450,8 @@ def _async_submit_note(self, sg, data):
         
         self.__upload_thumbnail(sg_note_data, sg, data)
         self.__upload_attachments(sg_note_data, sg, data)
-        
+
+        return sg_note_data
```

Comment:
if you could update the docstring for this method while you are changing the return value, that would be amazing.